GoogleForCreators/web-stories-wp
# Prepublish Checklist: Empty State

## Context

In testing for this next release a few times it's come up that the disabled state of prepublish high priority and recommended tabs is a little confusing. Since we're ditching it in the next round of PPC updates, figured we might as well eliminate the confusion before users see it. 

## Summary

- Remove the disable state of the simple panel that was added in last week (#7277) 
- Update empty component for checklist 
- Update logic for when empty component shows 

## Relevant Technical Choices

- Got a little creative to make sure the recommended tab stays present if all recommendations are handled but the optimize video experiment is true and not enabled yet. 
- Passing body text into empty component because I doubt that text is static for the different states - ie) unavailable at the start vs all the issues resolved. 
- Since there aren't any other PRs open for the checklist right now, took the opportunity to move some of the files in `/prepublish` to `prepublish/components/`so that structure's easier to parse through. 

## To-do

- ~Design's updating something, so do whatever visual update that is~ 

## User-facing changes

![Kapture 2021-05-04 at 14 33 07](https://user-images.githubusercontent.com/10720454/117073852-63663e80-ace7-11eb-99f6-03b94aa84a18.gif)


(Updated design, video above is still relevant for flow) 

When story has all issues resolved 
![Screen Shot 2021-05-05 at 1 37 15 PM](https://user-images.githubusercontent.com/10720454/117205968-23669080-ada7-11eb-9c48-62e8b4ee0e72.png)

Story at start
![Screen Shot 2021-05-05 at 1 37 36 PM](https://user-images.githubusercontent.com/10720454/117205971-23ff2700-ada7-11eb-8100-00bab2be3417.png)



## Testing Instructions

- This requires retesting the checkpoints of the prepublish checklist. 
- Initially on an empty story, you now see a message about how the checklist is empty. 
- As soon as you add another element (besides bg color) or a new page you should no longer see the empty message and you should see recommended panel. 
- When you reach a 5th page or when you try to early publish and instead go to review the checklist, you'll see the high priority panel as well. 
- If you clear out the entire checklist you'll get the empty checklist again. 
- If you only have the unchecked optimize video on upload you should still see that check in the recommended panel but there should be no badge (since there's 0 issues to resolve).




### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7375
